### PR TITLE
Fix shellcheck warnings in asv scripts

### PR DESCRIPTION
**Pull request by inducer**
@isuruf Could you take a look?

### CODE REVIEW

File: ci-support.sh [hunk 1]
```diff
@@ -550,7 +549,7 @@ function clone_results_repo
     else
       git clone https://gitlab.tiker.net/isuruf/benchmark-data
     fi
-    ln -s $PWD/benchmark-data/$PROJECT .asv/results
+    ln -s "$PWD/benchmark-data/$PROJECT .asv/results" .
```
Comment (isuruf) [inline ci-support.sh]:
```suggestion
    ln -s "$PWD/benchmark-data/$PROJECT" ".asv/results"
```

Comment (inducer) [reply ci-support.sh]: Thanks!

**Comment by inducer** [comment]
Test run with these changes: https://gitlab.tiker.net/inducer/sumpy/-/jobs/354387

**Comment by inducer** [comment]
Looks like it didn't break it. :tada: Thanks for your help!

**Pull request merged**